### DISCUSSION TITLE
ViewTube stopped working

### DISCUSSION THREAD

**Discussion by tbreitha** [ViewTube/viewtube#2666] [General]
Since this morning ViewTube stooped working for me.
I still can open the Website, but it is almost blank.
I'm getting Error loading homepage. The API may not be reachable.
Anyone else experiences the same issue?
![viewtube](https://github.com/ViewTube/viewtube/assets/100781904/78c57f3b-638d-4632-a864-53977fcb44c5)

**Comment by moisout** [comment]
Fixed with release 0.15.2 :)
